Content.Server/_Goobstation/StationEvents/Components/SelectedGameRulesComponent.cs

```diff
@@ -0,0 +1,17 @@
+using Content.Shared.EntityTable.EntitySelectors;
+
+namespace Content.Server._Goobstation.StationEvents.Components;
+
+
+/// <summary>
```

Comment:
Remove extra blank line
```suggestion
namespace Content.Server._Goobstation.StationEvents.Components;

/// <summary>
```